containers/app/app.py

```diff
@@ -0,0 +1,56 @@
+import os
+import subprocess
+import connexion
+import sqlalchemy
+from shapely.geometry import Polygon
+from shapely import wkt
+
+
+DB_PASSWORD = os.environ['POSTGRES_PASSWORD']
+RAW_DIR = '/osm/pbf'
+CONVERTED_DIR = '/osm/pbf/converted'
+MAPCONFIG_PATH = '/osm/mapconfig.xml'
+engine = sqlalchemy.create_engine(f'postgresql+psycopg2://postgres:{DB_PASSWORD}@pgrouting/postgres')
+
+
+class Loader:
+    @staticmethod
+    def create_dirs(directory):
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+
+    @staticmethod
+    def convert(raw_directory, converted_directory):
+        for file in os.listdir(raw_directory):
+            if file.endswith(".osm.pbf"):
+                path = os.path.join(raw_directory, file)
+                osm_name = file.rstrip('.pbf')
+                new_path = os.path.join(converted_directory, osm_name)
+                subprocess.check_call([f'osmconvert {path} --drop-author --drop-version --out-osm -o={new_path}'], shell=True)
```

Comment:
Again about the naming.
This name does not reveal what function do. User doesn't understand what is raw file and what is conwerted file.
I suggest: 
```suggestion
    @staticmethod
    def convert_pbf_to_osm(pbf_file_directory: str, osm_file_directory: str):
        for file in os.listdir(pbf_file_directory):
            if file.endswith(".osm.pbf"):
                path = os.path.join(pbf_file_directory, file)
                osm_name = file.rstrip('.pbf')
                new_path = os.path.join(osm_file_directory, osm_name)
                subprocess.check_call([f'osmconvert {path} --drop-author --drop-version --out-osm -o={new_path}'], shell=True)
```